src/Form/Control/TreeItemSelector.php

```diff
@@ -112,6 +120,46 @@ public function setTreeItems(array $treeItems)
         return $this;
     }
 
+    /**
+     * Recursive function to return sub-node for a given parent.
+     *
+     * @param $model : Model
```

Comment:
This is invalid phpdoc, was it created using some IDE?

If yes, fix your IDE asap, otherwise it may be merged by a mistake. Also make sure no phpdoc is added if not needed.